modules/filecheck/collect_dirs.go

```diff
@@ -59,6 +63,10 @@ func (fc *Filecheck) addDirToCharts(dirpath string) {
 			id = dirDimID(dirpath, "mtime_ago")
 		case dirNumOfFilesChart.ID:
 			id = dirDimID(dirpath, "num_of_files")
+		case dirNumOfFilesChart.ID:
+			id = dirDimID(dirpath, "num_of_files")
```

Comment:
A duplicate. Please remove it.

Reply:
hawk eye! 🐦

Reply:
removed in https://github.com/netdata/go.d.plugin/pull/487/commits/edc7b2f627bdd88d158468c04493ca3b13af7285